src/components/Lend/LoanSearch/LoanSearchFilterChips.vue

```diff
@@ -105,7 +146,58 @@ export default {
 			this.$emit('updated', formattedFacet);
 
 			this.$kvTrackEvent('Lending', 'click-remove-filter-chip', `${facetType}-${facet.name}`);
-		}
+		},
+		handleResetClick() {
+			this.$emit('reset');
+
+			this.isCollapsed = true;
+
+			this.$kvTrackEvent('Lending', 'click-reset-filter-chips', 'Loan Search Filter Chips Reset');
+		},
+		async determineIsCollapsable() {
+			// Refs will be undefined initially if Vue reused the component
+			if (!this.$refs.container || !this.$refs.chips) {
+				await this.$nextTick();
+			}
+
+			const { container, chips } = this.$refs;
+
+			if (container && chips) {
+				// Store whether collapsed to restore after calculation below
+				const storedIsCollapsed = this.isCollapsed;
+
+				// Collapse the chips and hide the toggle link
+				this.isCollapsed = true;
+				this.isCollapsable = undefined;
+
+				// Update virtual DOM with single row of chips
+				await this.$nextTick();
+
+				this.isCollapsable = chips.clientHeight > container.clientHeight;
+				this.isCollapsed = storedIsCollapsed;
+			}
+		},
+		handleToggleClick() {
+			this.isCollapsed = !this.isCollapsed;
+
+			this.$kvTrackEvent(
+				'Lending',
+				'click-toggle-filter-chips',
+				'Loan Search Filter Chips Toggled',
+				this.isCollapsed ? 'hidden' : 'shown'
+			);
+		},
 	},
+	watch: {
+		items() {
+			this.determineIsCollapsable();
+		}
+	}
 };
 </script>
+
+<style lang="postcss" scoped>
+	.container { max-height: 148px; }
```

Comment:
We may want to use `chip-container` here to be more specific... I don't think there are any other generic usages on this page of the `container` class but there may be elsewhere.